test/unit/testtest_spec.lua

```diff
@@ -0,0 +1,19 @@
+local helpers = require('test.unit.helpers')(after_each)
+local assert = require('luassert')
+
+local itp = helpers.gen_itp(it)
+
+local sc = helpers.sc
+
+-- All of the below tests must fail. Check how exactly they fail.
+if os.getenv('NVIM_TEST_RUN_TESTTEST') ~= '1' then
+  return
+end
```

Comment:
Amount of work is not the same at all:

1. I run tests in one terminal, develop in another. To run this file I need to get command from history with environment variable temporary set in the command, this is almost no work needed. Not the same if you do commenting/uncommenting: switch to another terminal, (un)comment, save, switch back, run.
2. Commenting/uncommenting is very prone to forgetting to uncomment back and make the change inside a commit.